client/signin.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import Signup from './signup.jsx';
+import ReactDOM from 'react-dom';
+
+function postSend(url, data) {
+  return fetch(url, {
+    credentials: 'same-origin', 
+    method: 'POST', // 'GET', 'PUT', 'DELETE', etc.
+    body: JSON.stringify(data), // Coordinate the body type with 'Content-Type'
+    headers: new Headers({
+      'Content-Type': 'application/json',
+      'Accept': 'application/json'
+    }),
+  }).then((result)=>{
+    ReactDOM.render(<Signup />, document.getElementById('app'));
+  })
+  .catch((error) =>{
+    console.error(error, 'postRequest error');
+  })
+};
+
+class Signin extends Component {
+  constructor(props){
+    super(props);
+    this.state = { id: "", password: ""}
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+  componentDidMount() {
+}
```

Comment:
closing bracket right after {

also add empty line before and after each function